pepper-apis/dss-database/src/main/resources/db-changes/schema/DDP-8554.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="Dmitrii Eliseev" id="2022-08-29-activity-validation-fields-size">
+        <modifyDataType columnName="expression_text"
+                        newDataType="TEXT"
+                        tableName="activity_validation" />
+
+        <modifyDataType columnName="precondition_text"
+                        newDataType="TEXT"
+                        tableName="activity_validation"/>
```

Comment:
This necessary?

Reply:
After the simplification of expression it's not, but anyway we're near the limit of the field so I'd prefer leaving that change.